container/root/etc/nginx/nginx.conf

```diff
@@ -0,0 +1,125 @@
+#############################################################
+# Replace nginx configuration directly in this file,
+# removing the need to perform error-prone replacements
+# at build time.
+#
+# For run-time replacements, ie, consuming environment vars,
+# add to the run.d/nginx script
+#############################################################
+
+user nobody;
+worker_processes auto;
+
+pid /tmp/nginx.pid;
+
+events {
+    worker_connections 1024;
```

Comment:
http://stackoverflow.com/questions/24318543/how-to-set-ulimit-file-descriptor-on-docker-container-the-image-tag-is-phusion

The file limits are actually from the host